NewRemarques.md

```diff
@@ -0,0 +1,12 @@
+Il faut réfléchir aux story
```

Comment:
There we have to make difference from insta I think. Perhaps all post from the day should go in story. What do you think?

Reply:
Why not.
Do we have pined post? Then user can pin a specific post to his profile?

Reply:
just to differentiate the service, make something a little bit different but there is no other reason. We don't have actually pined post but we could and perhaps should in some way in order to give them the ability to show something very specific on top of their profile.